brainscore_vision/data/bracci2019/requirements.txt

```diff
@@ -0,0 +1,5 @@
+natsort
+logging
+scipy
+numpy
+pathlib
```

Comment:
are these all needed for data *use* or only for data *packaging*? If it's only the later, no need to include the requirements here